### DISCUSSION TITLE
Software compiled with “fyne-cross windows” lacks the .exe suffix

### DISCUSSION THREAD

**Discussion by junmaqiang** [fyne-io/fyne#3672] [Q&A] [answered]
```
fyne-cross windows -arch=amd64,386 -name=Demo-build -app-version=0.0.1 -icon=logo.png -output=Demo --env GOPROXY=https://goproxy.io,direct
```

**Comment by andydotxyz** [comment, marked as answer]
You have specified the output to be "Demo". Try "-output Demo.exe". By default it will be a .exe extension but you override it.

**Reply by junmaqiang** [reply]
thank you

**Reply by junmaqiang** [reply]
However, I noticed that docker didn't add the.exe suffix to me after I gave the parameter --output=Demo

## Reference code
https://github.com/fyne-io/fyne-cross/blob/master/internal/command/windows.go#L63

**Reply by andydotxyz** [reply]
That applies to "name" and the setting of a default "output" if the user does not override it.
This is working as expected.

**Reply by junmaqiang** [reply]
OK, I know right now , thank you @andydotxyz
